### ISSUE TITLE
TensorflowEmitter has not supported operator [batch_normalization].

### ISSUE THREAD

**Issue by dhaase-de** [Microsoft/MMdnn#14]
This happened to me when converting a Caffe classification model using batch norm to TensorFlow. I used Python 3 and `MMdnn-0.1.1`, installed via `pip`.

Should be fixed by changing `mmdnn/conversion/caffe/mapper.py:193` from
`return Node.create('batch_normalization', **kwargs)` to
`return Node.create('BatchNorm', **kwargs)`.

**Comment by kitstar** [comment]
Fixed. But the batch normalization of caffe converter is not well tested. Glad if you can help with it. Thanks.
